docs/REVIEWING.md

```diff
@@ -36,6 +36,40 @@ following items should also be checked:
 - [ ] Does it comply with [component spec](specs/component.md)?
 - [ ] Does it comply with the [instrumentation spec](specs/instrumentation.md)?
 
+
+### Sink review checklist
```

Comment:
```suggestion
### Checklist- new sink
```

(something like that, think it should convey this is for a _new_ sink and not just any sink PR.